src/System.Runtime/tests/System/Security/SecurityAttributeTests.cs

```diff
@@ -0,0 +1,50 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Security;
+using Xunit;
+
+namespace System.Tests
+{
+#pragma warning disable 618
+    public static class SecurityAttributeTests
+    {
+        public static void InstantiateSecurityAttributes()
+        {
+            var t = new SecurityTreatAsSafeAttribute();
+            var u = new SecuritySafeCriticalAttribute();
+            var v = new SecurityTransparentAttribute();
+            var w = new SecurityCriticalAttribute();
+            var x = new SuppressUnmanagedCodeSecurityAttribute();
+            var y = new UnverifiableCodeAttribute();
+            var z = new AllowPartiallyTrustedCallersAttribute();
```

Comment:
Do any of these have properties for which we should assert their default values?